sql/pgwire/v3.go

```diff
@@ -622,8 +622,8 @@ func (c *v3Conn) sendResponse(resp sql.Response, formatCodes []formatCode, sendD
 		return c.sendCommandComplete(nil)
 	}
 	for _, result := range resp.Results {
-		if result.PErr != nil {
-			if err := c.sendError(result.PErr.String()); err != nil {
+		if result.Err != nil {
+			if err := c.sendError(result.Err.Error()); err != nil {
```

Comment:
We don't want to send a `roachpb.Error` back to the client, but the error needs to still be in structured form by the time it reaches https://github.com/cockroachdb/cockroach/blob/ec91640349279ef008d24cc49d80005a477122dc/sql/pgwire/v3.go#L601

If we want to introduce a sql-specific structured error then that would work too but it seems excessive to introduce a second structured error type.